ytfzf: add page (#7458)
tldr-pages/tldr: pages/linux/ytfzf.md

# ytfzf

> A POSIX script that helps you find and download videos and music.
> More information: <https://github.com/pystardust/ytfzf>.

> - Search videos in YouTube and display the thumbnails:

Reviewer: Perhaps this will read better?
```suggestion
- Search for videos on YouTube and display the thumbnails:
```

> `ytfzf -t {{search text}}`

Reviewer 2: We prefer long arguments
```suggestion
`ytfzf --show-thumbnails {{search text}}`
```

> - Play only audio of first item in loop:

Reviewer: Perhaps this will read better?
```suggestion
- Play only the audio of the first item in a loop:
```

> `ytfzf -mal {{search text}}`

Reviewer 2: We use `snake_case` for placeholders
```suggestion
`ytfzf --audio-only --auto-select --loop {{search_pattern}}`
```

- Download a video from history:

> `ytfzf -dH`

Reviewer 2: ```suggestion
`ytfzf --download --choose-from-history`
```

> - Download a video from history:

Reviewer: Perhaps this will read better?
```suggestion
- Download a video from the history:
```

- Play all the music found in one search:

> `ytfzf -mA {{search text}}`

Reviewer 2: ```suggestion
`ytfzf --audio-only --select-all {{search_pattern}}`
```

> - Play all the music found in one search:

Reviewer: Perhaps this will read better?
```suggestion
- Play all the music found in a search:
```

- See the trending videos in external menu:

> `ytfzf -TD {{search text}}`

Reviewer 2: ```suggestion
`ytfzf --trending --ext-menu {{search_pattern}}`
```

- Search PeerTube instead of YouTube:

> `ytfzf -P {{search text}}`

Reviewer 2: ```suggestion
`ytfzf --peertube {{search_pattern}}`
```

> - See the trending videos in external menu:

Reviewer: Perhaps this will read better?
```suggestion
- See the trending videos in an external menu:
```
